### PR TITLE
Meme pr of the year

### PR DESCRIPTION
**Pull request by TGEnigma**

### CODE REVIEW

File: rpcs3/Emu/Cell/PPUThread.cpp [hunk 1]
```diff
@@ -165,7 +165,9 @@ extern const ppu_decoder<ppu_interpreter_fast> g_ppu_interpreter_fast([](auto& t
 
 extern void ppu_initialize();
 extern void ppu_initialize(const ppu_module& info);
-static void ppu_initialize2(jit_compiler& jit, const ppu_module& module_part, const std::string& cache_path, const std::string& obj_name, u32 fragment_index, atomic_t<u32>&);
+
+static void ppu_initialize2(class jit_compiler& jit, const ppu_module& module_part, const std::string& cache_path, const std::string& obj_name, u32 fragment_index, atomic_t<u32>&);
+
```
Comment (elad335) [inline rpcs3/Emu/Cell/PPUThread.cpp]: remove extra lines

**Comment by JXPZ** [comment]
https://ci.appveyor.com/api/buildjobs/1pr8t4ejlrwxxfpl/artifacts/rpcs3-v0.0.4-2018-01-05-addae860_win64.zip Persona 5😍😍😍😍

**Comment by AniLeo** [comment]
STOP SPAMMING

**Comment by JXPZ** [comment]
ok

**Pull request merged**